src/chat/components/InputToolbar.tsx

```diff
@@ -55,6 +62,34 @@ const InputToolbar: React.FC<IInputToolbar> = ({
     iconStyle,
   ]);
 
+  const openGallery = useCallback(async () => {
+    const options: ImageLibraryOptions = {
+      mediaType: 'mixed',
+    };
+    const result: ImagePickerResponse = await launchImageLibrary(options);
+    if (
+      result &&
+      !result.didCancel &&
+      !result.errorCode &&
+      result.assets &&
+      result?.assets.length > 0
+    ) {
+      const file = result.assets[0];
+      if (file) {
+        const mediaType = file.type?.startsWith('image') ? 'photo' : 'video';
+        const extension = mediaType === 'photo' ? 'jpg' : 'mp4';
+        onSend?.(
+          {
+            type: mediaType,
+            path: file.uri ?? '',
+            extension: extension,
+          },
+          true
+        );
+      }
+    }
```

Comment:
Should add the code to throw an error if the if condition is not met

Reply:
added throw catch